readme.md

```diff
@@ -1,49 +1,130 @@
+- Jump to [installation instructions](#installation-and-usage).
+- Try out the Binder [demo notebook](https://mybinder.org/v2/gh/NREL/OpenOA/main?filepath=examples).
+- Read the [documentation](https://openoa.readthedocs.io/en/latest/).
+- Learn how to [contribute](contributing.md).
 
-The library is written around Pandas Data Frames, utilizing a flexible backend
-so that data loading, processing, and analysis could be performed using other libraries,
-such as Dask and Spark, in the future.
+This library provides a framework to study operational assessment (OA) methodologies that consume
+timeseries data from wind plants. Its development has been motivated by the National Renewable
+Energy Laboratory (NREL) Wind Plant Performance Prediction (WP3) Benchmarking (PRUF) project [^1], which
+developed an open source, baseline implementation of the 20-year annualized energy production (AEP)
+for plant-level performance assessment. It has grown to incororporate several more analyses, and a
+schema for time-series data from wind power plants.
 
-If you would like to try out the code before installation or simply explore the possibilities, please see our examples on [Binder](https://mybinder.org/v2/gh/NREL/OpenOA/develop_v3?filepath=examples).
+> **Warning**
+OpenOA is a research software library and is released under a BSD-3 license. Please refer to the
+accompanying [license file](LICENSE.txt) for the full terms. The inclusion of any algorithms and
+their implementations in the code or accompanying documentation **shal not** be interpreted as an
+endorsement by the authors, NREL, nor the US Government for fitness in any particular purpose or
+situation. While the authors believe the software may be useful for industry practitioners, we
+encourage caution, use of best practices, and engagement with trained professionals when performing
+any data analysis.
 
-If you use this software in your work, please cite our JOSS article with the following BibTex:
+### Included Analysis Methods
 
-```
-@article{Perr-Sauer2021,
-  doi = {10.21105/joss.02171},
-  url = {https://doi.org/10.21105/joss.02171},
-  year = {2021},
-  publisher = {The Open Journal},
-  volume = {6},
-  number = {58},
-  pages = {2171},
-  author = {Jordan Perr-Sauer and Mike Optis and Jason M. Fields and Nicola Bodini and Joseph C.Y. Lee and Austin Todd and Eric Simley and Robert Hammond and Caleb Phillips and Monte Lunacek and Travis Kemper and Lindy Williams and Anna Craig and Nathan Agarwal and Shawn Sheng and John Meissner},
-  title = {OpenOA: An Open-Source Codebase For Operational Analysis of Wind Farms},
-  journal = {Journal of Open Source Software}
-}
+| Name | Description | Citations |
+| --- | --- | --- |
+| ``MonteCarloAEP`` | This routine estimates the long-term annual energy production (AEP) of a wind power plant (typically over 10-20 years) based on operational data from a shorter period of record (e.g., 1-3 years), along with the uncertainty. | [^2], [^3] |
```

Comment:
Most of the PRUF publications describe the AEP analysis method, so we could also cite references 1 and 4 if you think that's appropriate here.